File: 2496. Maximum Value of a String in an Array/solution.py
class Solution(object):
  def maximumValue(self, strs):
    """
    :type strs: List[str]
    :rtype: int
    """
    result = 0
    for value in strs:
      currentCount = len(value)
      if value.isdigit():
        currentCount = int(value)
      if currentCount > result:
        result = currentCount
    return result

File: 2496. Maximum Value of a String in an Array/solution.js
/**
 * @param {string[]} strs
 * @return {number}
 */
var maximumValue = function(strs) {
  let result = 0;
  for (let index = 0; index < strs.length; index++){
    let currentCount = strs[index].length;
    if (!isNaN(strs[index])){
      currentCount = parseInt(strs[index]);
    }
    if (currentCount > result){
      result = currentCount;
    }
  }
  return result;
};